frontend/src/metabase/lib/i18n.js

```diff
@@ -32,6 +32,8 @@ export async function loadLocalization(locale) {
           },
         };
   setLocalization(translationsObject);
+
+  return translationsObject;
```

Comment:
This is used in `frontend/src/metabase/public/LocaleProvider.tsx`. Since `withInstanceLanguage` function resets the locale, but in static embeddings/public links, there is no user locale so `MetabaseUserLocalization` is set to the instance locale, so after the function has called, the locale that has been loaded by `loadLocalization` will be overridden by the instance locale.

This return value is used to set the `MetabaseUserLocalization` when necessary. For now it's only used in `LocaleProvider` like mentioned previously.